src/pages/Signup/Signup.js

```diff
@@ -1,7 +1,26 @@
 import React from 'react';
+import './Signup.scss';
 
 const Signup = () => {
-  return <div>Signup</div>;
+  return (
+    <main className="main-container">
```

Comment:
```suggestion
    <main className="signup">
```
가장 부모 태그의 className은 컴포넌트의 이름으로 구성해주세요!